tests/codebuild/run_canarytest.sh

```diff
@@ -98,12 +98,9 @@ pushd sagemaker-k8s-operator
 popd 
 
 echo "Waiting for controller pod to be Ready"
-kubectl \
-    wait \
-    --for=condition=Ready \
-    --timeout=5m \
-    "pods/$(kubectl get pods -n sagemaker-k8s-operator-system | grep sagemaker-k8s-operator-controller-manager | awk '{print $1}')" \
-    -n sagemaker-k8s-operator-system 
+# Wait to increase chance that pod is ready
+# Should upgrade kubectl to version that supports `kubectl wait pods --all`
```

Comment:
Lets just update `kubectl` then? We own the container that runs these tests

Reply:
Agreed, this change is to bring the canary `wait` in line with the integration test `wait` to fix failures. I'll make a task for this.

Reply:
How long would that take? If it's simply updating the version number inside a Dockerfile, it would be better to do that now and then change this PR to utilize the new CLI option rather than merge this?

Reply:
I am not confident that it will be that easy; I think there will have to be an existence check before waiting. I think this work should be prioritized against other optimization tasks for making tests faster.